graph/src/data/query/error.rs

```diff
@@ -273,6 +275,7 @@ impl fmt::Display for QueryExecutionError {
             Panic(msg) => write!(f, "panic processing query: {}", msg),
             EventStreamError => write!(f, "error in the subscription event stream"),
             FulltextQueryRequiresFilter => write!(f, "fulltext search queries can only use EntityFilter::Equal"),
+            FulltextQueryInvalidSyntax(msg) => write!(f, "Invalid fulltext search query syntax. msg: {}. Tip: Spaces are only allowed around operators", msg),
```

Comment:
I would leave out the `msg: {}.` part because that's the cryptic in-db details that we probably shouldn't expose to the client (database schema names etc.).

Reply:
Change to `Invalid fulltext search query syntax. Error: {}. Hint: Search terms with spaces need to be put in single quotes`?

Reply:
updated